Polyhedron/demo/Polyhedron/Plugins/AABB_tree/Cut_plugin.cpp

```diff
@@ -791,12 +791,13 @@ class Polyhedron_demo_cut_plugin :
   Q_INTERFACES(CGAL::Three::Polyhedron_demo_plugin_interface)
   Q_INTERFACES(CGAL::Three::Polyhedron_demo_io_plugin_interface)
   Q_PLUGIN_METADATA(IID "com.geometryfactory.PolyhedronDemo.PluginInterface/1.0")
+  Q_PLUGIN_METADATA(IID "com.geometryfactory.PolyhedronDemo.IOPluginInterface/1.90")
 
 public:
   Polyhedron_demo_cut_plugin() : QObject(), edges_item(0) {
   }
 
-  virtual ~Polyhedron_demo_cut_plugin();
+   ~Polyhedron_demo_cut_plugin();
```

Comment:
If you remove `virtual` you should take the opportunity to add `override` instead (at the end of the declaration).